spec/features/issues_spec.rb

```diff
@@ -158,7 +158,7 @@
       baz.save
       visit project_issues_path(project, sort: sort_value_oldest_updated)
 
-      first_issue.should include("baz")
+      expect(first_issue).to include("baz")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.